src/freenet/l10n/freenet.l10n.en.properties

```diff
@@ -1783,14 +1783,18 @@ SecurityLevels.enterPasswordTitle=Enter the master password to access cached con
 SecurityLevels.passwordWrong=You entered the wrong password. Please enter the correct password to enable access to the client-cache and improve browsing performance.
 SecurityLevels.enterPassword=Please enter the master password to enable access to downloads and uploads and to improve performance by enabling the recently browsed freesites cache.
 SecurityLevels.setPasswordLabel=Set the password
+SecurityLevels.confirmPasswordLabel=Confirm password
+SecurityLevels.passwordLabel=Password
 SecurityLevels.setPassword=Please set the master password, which will be used to protect cached content. The stronger the password the stronger the protection: lots of letters and numbers and punctuation is best.
 SecurityLevels.passwordNotZeroLength=The password must be at least one character long i.e. it must not be "".
+SecurityLevels.passwordsDoNotMatch=The passwords must match and they must be at least one character long i.e. it must not be "".
```

Comment:
Because the second message is used in cases where password confirmation is occurring and the passwords can be zero length or not match, hence extending the first message that is already used.